.changes/ndk-0.9.md

```diff
@@ -0,0 +1,7 @@
+---
+"tao": major
```

Comment:
```suggestion
"tao": minor
```

Reply:
Your documentation says to use `major`:

https://github.com/tauri-apps/tao/tree/HEAD/.changes

I cannot help the fact that the documentation doesn't understand what the most significant version is in semver, when major (or even minor) is still at `0`, you should fix that at the documentation level.

Reply:
yeah that file need to be updated, it was generated from the changes tool (`covector`) and never really had much refactors.